Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Video/GetAllVideosTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.MediatR.Media.Video.GetAll;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.Media.Video;
+
+public class GetAllVideosTest
+{
+    private Mock<IRepositoryWrapper> _mockRepository;
+    private Mock<IMapper> _mockMapper;
+
+    public GetAllVideosTest()
+    {
+        _mockRepository = new Mock<IRepositoryWrapper>();
+        _mockMapper = new Mock<IMapper>();
+    }
+
+    [Fact]
+    public async Task GetAllFacts_ShouldReturnSuccessfullyType()
+    {
+        //Act
+        _mockRepository.Setup(x => x.VideoRepository
+              .GetAllAsync(
+                  It.IsAny<Expression<Func<DAL.Entities.Media.Video, bool>>>(),
+                    It.IsAny<Func<IQueryable<DAL.Entities.Media.Video>,
```

Comment:
Why do you use full path here? Can we cut it down a little?

Reply:
Yea, got it. Also I`ll do it for Fact tests. Thanks.